src/midifallback.h

```diff
@@ -12,26 +12,30 @@
 // GNU General Public License for more details.
 //
 // DESCRIPTION:
-//      Windows native MIDI
-
-#ifndef __I_WINMUSIC__
-#define __I_WINMUSIC__
+//      MIDI instrument fallback support
```

Comment:
Why no header guards?

Reply:
Fixed in https://github.com/chocolate-doom/chocolate-doom/pull/1558/commits/b4dd44b62276098a2f6ce76e1ee3fd83db2ede34